frontend/packages/shipwright-plugin/src/utils.ts

```diff
@@ -0,0 +1,73 @@
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { getBuildRunStatusProps } from './components/buildrun-status/BuildRunStatus';
+import { BUILDRUN_TO_RESOURCE_MAP_LABEL } from './const';
+import { BuildRun, ComputedBuildRunStatus as runStatus } from './types';
```

Comment:
There is no need to rename this import:

```suggestion
import { BuildRun, ComputedBuildRunStatus } from './types';
```